error: XDG_RUNTIME_DIR not set in the environment.
error: XDG_RUNTIME_DIR not set in the environment.
Diameter: 17.0
TIME:  0.298470020294
CLOCKTIME:  9.777822
	Command being timed: "python /home/cpennycu/networkx/gt.py -f /home/cpennycu/gdata/er/eru-1-100000-2.snap -d 0"
	User time (seconds): 28.56
	System time (seconds): 0.38
	Percent of CPU this job got: 149%
	Elapsed (wall clock) time (h:mm:ss or m:ss): 0:19.40
	Average shared text size (kbytes): 0
	Average unshared data size (kbytes): 0
	Average stack size (kbytes): 0
	Average total size (kbytes): 0
	Maximum resident set size (kbytes): 102840
	Average resident set size (kbytes): 0
	Major (requiring I/O) page faults: 0
	Minor (reclaiming a frame) page faults: 39784
	Voluntary context switches: 337
	Involuntary context switches: 1510
	Swaps: 0
	File system inputs: 4608
	File system outputs: 16
	Socket messages sent: 0
	Socket messages received: 0
	Signals delivered: 0
	Page size (bytes): 4096
	Exit status: 0
